doc/dynamo0p3.rst

```diff
@@ -1256,6 +1336,27 @@ and at W1)::
        local_stencil, xdata, ydata, zdata, ndf_w0, undf_w0, map_w0, &
        basis_w0_on_w0, basis_w0_on_w1, ndf_w1)
 
+If the metadata specifies that the kernel requires a property of the
+reference element::
+
+  type, extends(kernel_type) :: testkern_operator_type
+     type(arg_type), dimension(2) :: meta_args =      &
+          (/ arg_type(gh_operator, gh_write, w0, w1), &
+             arg_type(gh_field*3, gh_read, w0) /)
+     type(reference_element_data_type) :: meta_reference_element(1) =  &
+          (/ reference_element_data_type(normal_to_face) /)
+     integer :: iterates_over = cells
+   contains
+     procedure, nopass :: code => testkern_operator_code
+  end type testkern_operator_type
+
+Then the kernel must be passed the number of horizontal faces of the
```

Comment:
Since we are defining which number of faces to pass in the metadata, I would suggest:
"Then the kernels must be passed the number of faces of the reference element as specified in the metadata (here horizontal) and the array of face normals::"